src/js/services/incomingData.js

```diff
@@ -84,27 +84,37 @@ angular.module('copayApp.services').factory('incomingData', function($log, $stat
 
     data = sanitizeUri(data);
 
-    // BIP21
-    if (bitcore.URI.isValid(data)) {
-      var parsed = new bitcore.URI(data);
+    // Bitcoin or Bitcoin Cash URL
+    if ((/^bitcoin[cash]*:/).exec(data)) {
+      var coin = 'btc';
+      if ((/^bitcoincash*:/).exec(data)) {
```

Comment:
This is also incorrect... this means "starts with bitcoincas and has zero to infinite "h" repetitions then a colon.
http://refiddle.com/refiddles/59b3575075622d0967780000

This should just be `/^bitcoincash:/`

Reply:
Updated. Thanks